frontend/src/metabase/dashboard/reducers.js

```diff
@@ -373,7 +281,6 @@ export const dashboardReducers = reduceReducers(
   INITIAL_DASHBOARD_STATE,
   combineReducers({
     ...typedReducers,
-    dashboards,
```

Comment:
are we not using this reducer?

Reply:
https://github.com/metabase/metabase/pull/48399/files#r1791522476 - it's the same answer